tests/TestCase/Shell/Task/TemplateTaskTest.php

```diff
@@ -446,6 +446,25 @@ public function testBakeIndex()
     }
 
     /**
+     * test bake template with index limit overwrite
+     *
+     * @return void
+     */
+    public function testBakeIndexWithIndexLimit()
+    {
+        $this->Task->controllerName = 'TemplateTaskComments';
+        $this->Task->modelName = 'TemplateTaskComments';
+        $this->Task->controllerClass = __NAMESPACE__ . '\TemplateTaskCommentsController';
+        $this->Task->params['indexColumns'] = 3;
```

Comment:
@markstory It happens here : https://github.com/DSchalla/bake/blob/index-columns/src/Shell/Task/TemplateTask.php#L405-L407